### ISSUE TITLE
Player can't see token they own.

### ISSUE THREAD

**Issue by adventuremagic123** [melek/lib_elevation#3]
I'm not sure what the problem is.  The player owns this token but it doesn't show up in their view.

Here, I'm running the server as a GM and "Test Player" connects to it.

![Screenshot 2021-08-21 155445](https://user-images.githubusercontent.com/50708405/130334694-38be087d-acc3-40a6-a6ce-369f573a9b4a.jpg)

**Comment by adventuremagic123** [comment]
I've removed the map so I don't get my hand slapped for copyright.  I'll markup a map distributed with MapTool in the future.

What I did was start over with a clean environment in that I deleted "Map Elevation Data" and used "Add Elev" to recreate it.

Here's the same thing with the map image removed (rename from *.zip to *.cmpgn):

[lib elevation test - issue #3.zip](https://github.com/melek/lib_elevation/files/7026217/lib.elevation.test.-.issue.3.zip)

![Screenshot 2021-08-21 160850](https://user-images.githubusercontent.com/50708405/130334833-90dd16f0-a217-4d26-9145-1677c35806a3.jpg)
